samples/04-file-transfer/README.md

```diff
@@ -130,35 +180,96 @@ java -Dedc.fs.config=samples/04-file-transfer/consumer/config.properties -jar sa
 java -Dedc.fs.config=samples/04-file-transfer/provider/config.properties -jar samples/04-file-transfer/provider/build/libs/provider.jar
 ````
 
-Assuming you didn't change the config files, the consumer will listen on port `9191` and the provider will listen on
-port `8181`. Open another terminal window (or any REST client of your choice) and execute the following REST request:
+Assuming you didn't change the ports in config files, the consumer will listen on port `9191`
+and the provider will listen on port `8181`.
+
+### 2. Initiate a contract negotiation
+
+In order to request any data, a contract agreement between provider and consumer is required.
+The provider has stated a contract offer for the file, but the consumer has yet to accept this offer.
+
+The consumer now needs to initiate a contract negotiation sequence with the provider. That sequence
+looks as follows:
+
+1. Consumer sends a contract offer to the provider (__currently, this has to be equal to the provider's offer!__)
```

Comment:
maybe we should mention, that the provider offers all of his assets in the form of contract offers, which are the basis for negotiation